lib/db/mem.js

```diff
@@ -732,6 +733,44 @@ module.exports = function (log, error) {
     return P.resolve({})
   }
 
+  // VERIFICATION REMINDERS
+
+  Memory.prototype.createVerificationReminder = function (request) {
+    var reminderData = request.body
+    if (! reminderData || ! reminderData.uid || ! reminderData.type) {
+      throw new Error('"uid", "type" are required')
+    }
+
+    reminderData.createdAt = Date.now()
+    reminders[reminderData.uid + reminderData.type] = reminderData
+
+    return P.resolve({})
+  }
+
+  Memory.prototype.fetchReminders = function (request) {
+    var query = request.query
+    if (! query || ! query.reminderTime || ! query.type || ! query.limit) {
+      throw error.wrap(new Error('fetchReminders - reminderTime, limit or type missing'))
+    }
+
+    var result = Object.keys(reminders)
+      .map(function (key) {
+        return reminders[key]
+      })
+      .filter(function (item) {
+        return item.type === query.type && (Date.now() - item.createdAt) > query.reminderTime
+      })
+      .slice(0, query.limit)
+
+    return P.resolve(result)
+  }
+
+  Memory.prototype.deleteReminder = function (reminderData) {
+    delete reminders[reminderData.email + reminderData.type]
```

Comment:
Should this be `reminders[reminderData.uid + reminderData.type]` so that it matches from create reminder?